.github/workflows/xahau-ga-macos.yml

```diff
@@ -76,6 +76,9 @@ jobs:
           max_size: 2G
           hash_dir: true
           compiler_check: content
+          is_main_branch: ${{ github.ref_name == env.MAIN_BRANCH_NAME }}
+          main_cache_dir: ~/.ccache-main
+          current_cache_dir: ~/.ccache-current
```

Comment:
Isn't it unnecessary to specify this because it is the same as the default value?

Reply:
Yes, good point, updated